site/content/docs/09-building.md

```diff
@@ -12,4 +12,21 @@ The output is a Node app that you can run from the project root:
 
 ```bash
 node __sapper__/build
-```
\ No newline at end of file
+```
+
+### Browser support
+
+Your site is built only for the latest versions of modern evergreen browsers by default. If you are using Rollup, you can use the `--legacy`[1] flag to build a second bundle for legacy browsers like Internet Explorer. Sapper will then serve up the correct bundle at runtime[2].
```

Comment:
Maybe ins tead of:
```
If you are using Rollup, you can use the `--legacy`[1] flag to build a second bundle for legacy browsers like Internet Explorer.
```

We could try:

```
If you are using Rollup, you can use the `--legacy`[1] flag to build a second bundle that can be used to support legacy browsers like Internet Explorer.
```

Hinting at its purpose rather than suggesting it solves that particular problem.

Reply:
Sure. The suggested language sounds fine to me. Updated